core/primitives/res/runtime_configs/parameters.txt

```diff
@@ -94,9 +94,8 @@ wasm_keccak512_byte: 36_649_701
 wasm_ripemd160_base: 853_675_086
 wasm_ripemd160_block: 680_107_584
 wasm_ecrecover_base: 3_365_369_625_000
-# both ed25519_verify_base and ed25519_verify_byte have NON-FINAL numbers (needs fine tuning)
-wasm_ed25519_verify_base: 40_311_888_867
-wasm_ed25519_verify_byte: 423_978_605
+wasm_ed25519_verify_base: 210_000_000_000
+wasm_ed25519_verify_byte: 9_000_000
```

Comment:
Hm, taking a second look here...

Is this the right file? Or should we add it at the protocol version where we actually enable the fn? 

It seems we do this in two different way, eg

https://github.com/jakmeier/nearcore/blob/e57229ecc713a68a252041d0dd3173d40a318417/core/primitives/res/runtime_configs/53.txt#L2

adds a new cost for a specific version.

That's mostly unrelated to the PR, I guess either one works (right?), but maybe it makes sense to unify. 

Or to, like, reverase the dircetion of diff already :)

Reply:
Hehe, I was hoping to sweep this inconsistency under the carpet :P

Truth is, I am not 100% decided which option is better but both already have precedence. I think adding it to parameters.txt is better, simply because otherwise we run into problems deserializing the intermediate JSON back to `ExtCostsConfig`. (That's why I suggested it like this in the nearcore guide book.)

In the case you mentioned we worked around it with serde default:
```rust
/// Cost for reading trie node from memory
#[serde(default = "default_read_cached_trie_node")]
pub read_cached_trie_node: Gas,
```

Reversing the diffs would be great, really. But I don't see myself doing it soon.
Unifying could be an option, I think there are not that many cases yet.

Reply:
let this slip then!